eZ/Bundle/EzPublishCoreBundle/EzPublishCoreBundle.php

```diff
@@ -84,6 +86,8 @@ public function build(ContainerBuilder $container)
         $container->addCompilerPass(new PlaceholderProviderPass());
         $container->addCompilerPass(new NotificationRendererPass());
         $container->addCompilerPass(new ConsoleCacheWarmupPass());
+        $container->addCompilerPass(new LazyDoctrineRepositoriesPass());
```

Comment:
I think that this CompilerPass should be done after all the "normal" ones, since it's purpose is to validate the service definitions. How about moving it to:
```suggestion
        $container->addCompilerPass(new LazyDoctrineRepositoriesPass(), PassConfig::TYPE_BEFORE_REMOVING
);
```
?